dataland-frontend/tests/e2e/specs/index.ts

```diff
@@ -55,6 +55,5 @@ if (testGroupingDisabled || cypressTestGroup === 3) {
 
 if (testGroupingDisabled || cypressTestGroup === 4) {
   require("./search-company-framework-data");
-  require("./data-request");
```

Comment:
There will be probably a new test concerning the frontend page, i assume.

Reply:
pushed
![image](https://github.com/d-fine/Dataland/assets/96523011/10c5fe0f-5042-4f29-b991-0657878d1dae)

Reply:
ok, i will look into the tests